lib/giftbit/base.rb

```diff
@@ -0,0 +1,79 @@
+require 'active_support'
+
+begin
+  require 'rest-client'
+rescue LoadError
+end
+
+module Giftbit
+  module Base
+    def self.included(klass)
+      klass.extend ClassMethods
+
+      # By default, set the endpoint to production
+      klass.endpoint  ||= 'https://api.giftbit.com/papi/v1/'
```

Comment:
The previous version set it to the test endpoint by default, which I think makes more sense. That way you don't accidentally create real gift cards when trying out the gem for the first time.